aptible/resource_database_test.go

```diff
@@ -247,10 +295,12 @@ func testAccAptibleDatabaseVersion(envId int64, dbHandle string) string {
 func testAccAptibleDatabaseUpdate(envId int64, dbHandle string) string {
 	return fmt.Sprintf(`
 	resource "aptible_database" "test" {
-    env_id = %d
+    	env_id = %d
```

Comment:
🤦 someone previously used spaces instead of tabs. If you could get this file using tabs, that'd be great but it isn't necessary. `resource_app_test.go` has the same problem when you modify the app resource.